front/src/RefboxField/RefboxButton.js

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import styled from "styled-components";
+import PropTypes from "prop-types";
+
+/**
+ * A button part of the refbox. Does a callback to its parent to handle onClick events
+ * Author: S.A. Tanja
+ * Author: G.W. van der Heijden
+ *
+ * Props:
+ *  - teamColor: the team color of the button (cyan or magenta).
+ *  - onClick: a function on what to do when the button is pressed
+ *  - children: the children of the button
+ */
+const RefboxButton = props => {
```

Comment:
Why do we need yet another button class? Can we not reuse TurtleEnableButton or something?

Reply:
Because this button is not a turtle button. You could make a individual button class like dropdown.